ci/before_script_linux.sh

```diff
@@ -1,22 +0,0 @@
-set -ex
```

Comment:
Do we want to keep the script, in case we need to re-generate the fixtures?

Reply:
Don't think it's necessary but sure